content/How to use Lottie Animations on SwiftUI.md

```diff
@@ -0,0 +1,64 @@
+![Main image](images/lottie-cover.png)
+
+# How to use Lottie Animations in SwiftUI
+
+## Introduction
+
+If you're here already, there's a 99.999% chance you know what Lottie is. For those still confused on what this article is about: Lottie is an awesome iOS, Android, and React Native library created by Airbnb that renders After Effects animations in real time, allowing you to use animations as easily as you use static images. You can find more info on their [website](https://airbnb.design/lottie/) and all the technical documentation you want on their [github](https://github.com/airbnb/lottie) page.
+
+
+## Overview
+
+In this tutorial, we’ll create a simple SwiftUI view using `UIViewRepresentable` in order to create and load any Lottie animation into your project.
+
+## Lets get started
+
+### Installation
+
+In order to use Lottie inside your project we need to install the library. For this we have a few different options like Cocapods, Carthage or the one we will use in this tutorial Swift Package Manager.
+
+First of all, on Xcode click on `File` and then select the `Add Package Dependency...` option in Swift Packages. In the input box, enter: `https://github.com/airbnb/lottie-ios.git` (this is the library repository from where Xcode will download the code) and then just hit next and leave the major version thats selected by default to use the more recent release for Lottie.
+
+
+### Hosting Lottie on SwiftUI 
+
+Now that you have Lottie already installed on your project we can start creating the view. For this you'll need to create a new SwiftUI file. Call it `LottieView` and delete everything except the import statement and add the following:
+
+https://gist.github.com/pMalvasio/93a02c5fb8d2f35509d78df8571915cd
+
+So far we're using a `UIViewRepresentable` to allocate an empty `UIView` with frame `.zero`.
+
+
+### Loading the Animation
+
+Next, you'll need to import Lottie on your code, so yo can add `import Lottie` below the SwiftUI import that's already there. On the `makeUIView(...)` method you can now call Lottie and load an animation and add it to the view we already created.
+
+https://gist.github.com/pMalvasio/82ec0abbdcf9b1776a533e673e8f6533
+
+
+### Adding the constraints
+
+To ensure that it takes the full height and width of the container, we'll specify constraints. To do so, add the following to before returning the view:
+
+https://gist.github.com/pMalvasio/06844e05841cc42cc26797e9e6d38ff1
+
+
+### Almost there
+
+The last part will be allow the `LottieView` to load any animation you import into your project. To do it we can add a new var and pass it at the moment of calling the animation inside the `makeUIView(...)` method:
```

Comment:
```suggestion
The last part will be to allow the `LottieView` to load any animation you import into your project. To do it we can add a new var and pass it at the moment of calling the animation inside the `makeUIView(...)` method:
```

Reply:
I know, the to I made you delete in the last comment add it here, just to restore balance 😂

Reply:
Balanced, as all things should be :thanos-snap: